Sanitizer error in ydb/tests/solomon/reading/basic_reading.py.TestBasicReading.test_basic_reading_monitoring

ydb/tests/solomon/reading/basic_reading.py.TestBasicReading.test_basic_reading_monitoring

**Build Information:**
- Version: 07893ec125f3264a8419172497302d7b5e97e7e5
- Build Preset: release-msan
- Branch: stable-25-4-1

**Run Command:**
```
./ya make -ttt --build "release" --sanitize="memory" -DDEBUGINFO_LINES_ONLY -F 'basic_reading.py::TestBasicReading::test_basic_reading_monitoring' ydb/tests/solomon/reading
```

**Owner:** [TEAM:@ydb-platform/fq](https://github.com/orgs/ydb-platform/teams/fq)

**Test run history:** [link](https://datalens.yandex/34xnbsom67hcq?branch=stable-25-4-1&full_name=ydb/tests/solomon/reading/basic_reading.py.TestBasicReading.test_basic_reading_monitoring&build_type=release-msan)

More info in [dashboard](https://datalens.yandex/4un3zdm0zcnyr)

**Logs:**
- [log](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_Small_and_Medium/20993149698/ya-stable-25-4-1-x86-64-msan/try_1/artifacts/logs/ydb/tests/solomon/reading/test-results/py3test/testing_out_stuff/basic_reading.py.TestBasicReading.test_basic_reading_monitoring.log)
- [logsdir](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_Small_and_Medium/20993149698/ya-stable-25-4-1-x86-64-msan/try_1/test_artifacts/ydb-tests-solomon-reading.zip)

**Error Summary:**
```
E   ydb.tests.library.harness.daemon.SeveralDaemonErrors: Daemon failed with message: Unexpectedly finished before stop.
E   Process exit_code = 100.
E   Stdout file name: 
E   /home/runner/actions_runner/_work/ydb/ydb/tmp/out/ydb/tests/solomon/reading/test-results/py3test/testing_out_stuff/basic_reading.py.TestBasicReading.test_basic_reading_solomon/cluster/node_1/stdout
E   Stderr file name: 
E   /home/runner/actions_runner/_work/ydb/ydb/tmp/out/ydb/tests/solomon/reading/test-results/py3test/testing_out_stuff/basic_reading.py.TestBasicReading.test_basic_reading_solomon/cluster/node_1/stderr
E   Stderr content:
E   
E   GRpc memory quota was set but disabled due to issues with grpc quoter, to enable it use EnableGRpcMemoryQuota option
E   ==475519==WARNING: MemorySanitizer: use-of-uninitialized-value
E   warning: address range table at offset 0x820 has a premature terminator entry at offset 0x830
E   #0 0x00004bd862a0 in NYql::NDq::(anonymous namespace)::TDqSolomonMetricsQueueActor::AnswerPendingRequests(bool) /-S/ydb/library/yql/providers/solomon/actors/dq_solomon_metrics_queue.cpp:378:9
E   #1 0x00004bd77f76 in HandleNextMetricsListingChunkReceived /-S/ydb/library/yql/providers/solomon/actors/dq_solomon_metrics_queue.cpp:266:9
E   #2 0x00004bd77f76 in NYql::NDq::(anonymous namespace)::TDqSolomonMetricsQueueActor::ThereAreMetricsToListState(TAutoPtr<NActors::IEventHandle, TDelete>&) /-S/ydb/library/yql/providers/solomon/actors/dq_solomon_metrics_queue.cpp:116:17
E   #3 0x00001c314fd2 in NActors::IActor::Receive(TAutoPtr<NActors::IEventHandle, TDelete>&) /-S/ydb/library/actors/core/actor.cpp:406:17
E   #4 0x00001c3c2ad8 in NActors::TExecutorThread::Execute(NActors::TMailbox*, bool) /-S/ydb/library/actors/core/executor_thread.cpp:268:28
E   #5 0x00001c3cd3b5 in NActors::TExecutorThread::ProcessExecutorPool()::$_0::operator()(NActors::TMailbox*, bool) const /-S/ydb/library/actors/core/executor_thread.cpp:458:39
E   #6 0x00001c3cc9fc in NActors::TExecutorThread::ProcessExecutorPool() /-S/ydb/library/actors/core/executor_thread.cpp:510:13
E   #7 0x00001c3ce932 in NActors::TExecutorThread::ThreadProc() /-S/ydb/library/actors/core/executor_thread.cpp:536:9
E   #8 0x000019d4c373 in (anonymous namespace)::TPosixThread::ThreadProxy(void*) /-S/util/system/thread.cpp:245:20
E   #9 0x7fc0b3031ac2  (/lib/x86_64-linux-gnu/libc.so.6+0x94ac2) (BuildId: 4f7b0c955c3d81d7cac1501a2498b69d1d82bfe7)
E   #10 0x7fc0b30c38bf  (/lib/x86_64-linux-gnu/libc.so.6+0x1268bf) (BuildId: 4f7b0c955c3d81d7cac1501a2498b69d1d82bfe7)
E   
E   SUMMARY: MemorySanitizer: use-of-uninitialized-value /-S/ydb/library/yql/providers/solomon/actors/dq_solomon_metrics_queue.cpp:378:9 in NYql::NDq::(anonymous namespace)::TDqSolomonMetricsQueueActor::AnswerPendingRequests(bool)
E   Exiting
```

--- a/ydb/library/yql/providers/solomon/actors/dq_solomon_metrics_queue.cpp
+++ b/ydb/library/yql/providers/solomon/actors/dq_solomon_metrics_queue.cpp
@@ -490,7 +490,7 @@ private:
     THashSet<NActors::TActorId> FinishedConsumers;
     THashMap<NActors::TActorId, ui64> FinishingConsumerToLastSeqNo;
 
-    bool HasPendingRequests;
+    bool HasPendingRequests = false;
     THashMap<NActors::TActorId, TDeque<NDqProto::TMessageTransportMeta>> PendingRequests;
     std::vector<NSo::TSelectors> PendingLabelRequests;
     std::vector<NSo::TSelectors> PendingListingRequests;

